### PR TITLE
Feat: display deployed version

### LINKED ISSUE #44: [FEATURE] Display deployed version
### Description

It would be useful to know what version is deployed, similar to our other apps

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by jalezi**
Display deployed version in upper right corner of the map.

## Fixes Issue

Closes #44 

## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot 2022-10-09 at 21 22 43

## Note to reviewers

My prettier settings are different.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/EddieHubLive/pull/48"></a>

### CODE REVIEW

**Review by E-fais — APPROVED** [review]
LGTM :)

File: src/package.alias.json [hunk 1]
```diff
@@ -0,0 +1 @@
+../package.json
```
Comment (eddiejaoude) [inline src/package.alias.json:1]: not sure why this is needed?

File: src/components/map.js [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
+import { MapContainer, TileLayer, Marker, Popup } from 'react-leaflet';
+import * as L from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+import Version from './version';
 
 function Map({ events }) {
   return (
     <MapContainer
-      style={{ height: "100vh" }}
+      style={{ height: '100vh' }}
```
Comment (eddiejaoude) [inline src/components/map.js:9]: hmm is it possible to only change the lines you require and leave everything else as it is for the moment?

We should add these style configs to the project, that will be in another issue / pull request

**Comment by eddiejaoude** [comment]
Looks good, just a few small comments

File: src/package.alias.json (hunk 1 above)
Comment (jalezi) [reply src/package.alias.json]: No. There is better solution. 😄

File: src/components/map.js (hunk 2 above)
Comment (jalezi) [reply src/components/map.js]: Done.

**Review by eddiejaoude — APPROVED** [review]
Thanks 👍

**Pull request merged**